main.py

```diff
@@ -6,9 +6,24 @@
 from model.feasibility_model import FeasibilityModel
 from model.optimality_model import OptimalityModel
 from model.construction_model import ConstructionModel
+from model.shift_design_model import ShiftDesignModel
 from results.converter import Converter
 
 
+def run_shift_design_model(problem="rproblem5"):
+    """
+    Runs the shift design model.
+
+    :param problem: the problem instance to run.
+    :return: the solved model instance.
+    """
+
+    esp = ShiftDesignModel(name="esp_shift_design", problem=problem)
```

Comment:
Hadde gitt et nytt navn her. `esp` er en forkortelse for Employee Scheduling Problem